eslint.config.mjs

```diff
@@ -0,0 +1,59 @@
+import Codex from "eslint-config-codex";
+import { plugin as TsPlugin, parser as TsParser } from 'typescript-eslint';
+
+export default [
+    ...Codex,
+    {
+        files: ['src/**/*.ts'],
+        languageOptions: {
+            parser: TsParser,
+            parserOptions: {
+                project: './tsconfig.json',
+                tsconfigRootDir: './',
+                sourceType: 'module',
+            },
+        },
+        rules: {
+            'n/no-missing-import': ['off'],
+            'n/no-unsupported-features/node-builtins': ['off'],
+            'jsdoc/require-returns-description': ['off'],
+            'jsdoc/require-jsdoc': [
+                'error',
+                {
+                    'require': {
+                        'FunctionDeclaration': true,
+                        'MethodDefinition': true,
+                        'ClassDeclaration': true,
+                        'ArrowFunctionExpression': false,
+                        'FunctionExpression': false,
+                    }
+                }
+            ],
+            '@typescript-eslint/explicit-member-accessibility': ['off'],
+            '@typescript-eslint/naming-convention': [
```

Comment:
and here

Reply:
Even this naming convention rule, it is from the same reference I have taken from.

https://github.com/editor-js/image/blob/master/eslint.config.mjs

Reply:
The reason is only for allowing a leading underscore?

Reply:
yes for underscore and for the variables to follow `camelCase`